SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -197,6 +197,19 @@ Interest InterestFactory::CreateRoutingWorkerInterest()
 	return ServerInterest;
 }
 
+Interest InterestFactory::CreateStrategyWorkerInterest()
+{
+	Interest ServerInterest;
+	Query ServerQuery;
+
+	ServerQuery.ResultComponentIds = { SpatialConstants::STRATEGYWORKER_TAG_COMPONENT_ID };
```

Comment:
I think the strategy worker should also have interest in LB_TAG_COMPONENT_ID and 50 (Schema WellKnownComponentSet, not sure we have a constant for it yet)

Reply:
Done.